oidc-controller/api/test_signing_key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Same as other `test_sigining_key.pem` file: if it is required for tests (and it looks like it may be the same file) we will want it in the `tests` folder. Even better: could the framework generate its own keyfile? If that is the case, these files should be gitignored.

Reply:
this is me in the middle of trying to fix an issue. sorry for the confusion, see above for the issue